solr/Dockerfile

```
@@ -1,5 +1,6 @@
 #ddev-generated
-FROM solr:9.6
+ARG SOLR_BASE_IMAGE="solr:9.6"
```

Use `scratch` to maintain this value only in one place:

```suggestion
ARG SOLR_BASE_IMAGE="scratch"
```